src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,22 @@
+.stars {
+  display: flex;
+  align-items: center;
+
+  @for $i from 1 through 5 {
+    &--#{$i} &__star:nth-child(-n + #{$i}) {
+      background-image: url(../images/star-active.svg);
+    }
+  }
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), when using SCSS loops, it's recommended to include a newline after the opening brace `{` and before the closing brace `}` to improve readability.